basics/concepts.md

```diff
@@ -1,79 +1,82 @@
+Multitenancy is an optional feature in Pinot. All tables, brokers, and servers belong to a tenant called _DefaultTenant_ in the absence of explicit tenant configuration.
 
-### **Cluster**
+### Cluster
 
-Logically, a [cluster ](components/cluster/)is simply a group of tenants. As with the classical definition of a cluster, it is also a grouping of a set of compute nodes. Typically, there is only one cluster per environment/data center. There is no needed to create multiple clusters since Pinot supports the concept of tenants. At LinkedIn, the largest Pinot cluster consists of 1000+ nodes distributed across a data center. The number of nodes in a cluster can be added in a way that will linearly increase performance and availability of queries. The number of nodes and the compute resources per node will reliably predict the QPS for a Pinot cluster, and as such, capacity planning can be easily achieved using SLAs that assert performance expectations for end-user applications.&#x20;
+A Pinot [_cluster_](components/cluster/) is a complete collection of the software processes and hardware resources required for Pinot to ingest data, store it, and process queries. The components of a cluster are described in more detail under _Physical Architecture_ below.
 
-{% hint style="info" %}
-Auto-scaling is also achievable, however, we recommend a set amount of nodes to keep QPS consistent when query loads vary in sudden unpredictable end-user usage scenarios.
-{% endhint %}
 
-## Pinot components
+## Physical Architecture
 
 ![](../.gitbook/assets/Pinot-Components.svg)
 
-A Pinot cluster consists of multiple distributed system components. These components are useful to understand for operators that are monitoring system usage or are debugging an issue with a cluster deployment.
+A Pinot cluster consists of the following processes, which are typically deployed on separate hardware resources in production. In development, they can fit comfortably into Docker containers on a typical laptop.
 
-* Controller
-* Broker
-* Server
-* Minion (optional)
+* **Controller**. Maintains cluster metadata and orchestrates cluster resources.
+* **Zookeeper**. Fault-tolerant, persistent storage on behalf of the controller.
+* **Broker**. Accepts queries from client processes and forwards them to servers for processing.
+* **Server**. Provides storage for segment files and compute for query processing.
+* **Minion** (optional). Distributed compute for cluster tasks other than query processing.
```

Comment:
```suggestion
* **Controller**: Maintains cluster metadata and orchestrates cluster resources.
* **Zookeeper**: Fault-tolerant, persistent storage on behalf of the controller.
* **Broker**: Accepts queries from client processes and forwards them to servers for processing.
* **Server**: Provides storage for segment files and compute for query processing.
* (Optional) **Minion**: Distributed compute for cluster tasks other than query processing.
```

Reply:
what does orchestrate mean in this context?

Reply:
...wondering if we can make this list parallel
would "Computes cluster tasks other than query processing" change the meaning?
same: Persists fault-tolerant storage on behalf of the controller